hadoop-ozone/common/src/main/java/org/apache/hadoop/ozone/OmUtils.java

```diff
@@ -279,6 +279,8 @@ public static boolean isReadOnly(
     case PrintCompactionLogDag:
     case GetSnapshotInfo:
     case GetServerDefaults:
+    case GetQuotaRepairStatus:
+    case StartQuotaRepair:
```

Comment:
This StartQuotaRepair is a write operation.

Reply:
It trigger a repair activity, but not directly a write operation to ratis --> and as part of this, it further trigger a write operation to ratis based on repair to be done.
Since can not be submitted to Ratis as no direct write action, so path of Read is followed.
This is similar to "RangerBGSync", "SnapshotDiff" where trigger happens.

Reply:
I see, thanks for the explanation.